src/components/ListPage.js

```diff
@@ -2,32 +2,36 @@ import React from "react"
 import { Link } from "react-router-dom"
 
 const ListPage = props => {
-  const { img_url, name, vaccination_status, age, id } = props.data
+	const { img_url, name, vaccination_status, age, id } = props.data
 
-  return (
-    <div className="row add-pets-section">
-      <div className="small-12 medium-6 columns about-pets">
-        <div className="about-pets-avatar">
-          <img className="avatar-image" src={img_url} alt={name} />
-        </div>
-        <div className="about-pets-author">
-          <p className="author-name">{name}</p>
-          <p className="author-location">Age: {age}</p>
-          <p className="author-mutual">
-            <strong>Up to Date on Shots?</strong>{" "}
-            {vaccination_status == true ? "Yes" : "No"}
-          </p>
-        </div>
-      </div>
-      <div className="small-12 medium-6 columns add-friend">
-        <div className="add-friend-action">
-          <Link to={`/pets/${id}`}>
-            <button className="button primary small">Adopt Me</button>
-          </Link>
-        </div>
-      </div>
-    </div>
-  )
+	return (
```

Comment:
You went in a little too far. 
I would suggest backing up the code on line 5 to be even with the line above it, but put a line break between the two. 
Then you can line the return up with that line. From there you would keep the first div even with the return, and indent each line appropriately from there. If you need a hand with that check with your peers, or put a question in the queue.